chart/values.yaml

```diff
@@ -87,6 +87,7 @@ injector:
     repo: chaos-injector
   annotations: {} # extra annotations passed to the chaos injector pods
   labels: {} # extra labels passed to the chaos injector pods
+  tolerations: [] # extra tolerations passed to the chaos injector pods
```

Comment:
I'm truly unsure, but should this be `tolerations : {}`, since we're checking `.key` and `.value` in the chart?

Reply:
I think we will still want an array because someone might want to add multiple tolerations.
If we left it as a map it would only support 1 key.

Checkout the kube docs here you can see its a list!
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#:~:text=has%20two%20tolerations%3A-,tolerations%3A,effect%3A%20%22NoExecute%22,-In%20this%20case